app/models/billing-detail.js

```diff
@@ -59,13 +62,28 @@ export default DS.Model.extend({
   planRate: Ember.computed('plan', function() {
     return PLAN_RATES[this.get('plan')];
   }),
-  containerAllowance: Ember.computed('plan', function() {
-    return CONTAINER_ALLOWANCES[this.get('plan')];
+
+  containersInPlan: Ember.computed('containerAllowance', 'plan', function() {
+    var plan = this.get('plan');
+    if (plan === 'development' || !CONTAINER_ALLOWANCES.hasOwnProperty(plan)) {
+      return 0;
+    }
+    return this.get('containerAllowance') || CONTAINER_ALLOWANCES[plan];
```

Comment:
Is the `development` assumption correct here or are there cases where an account could have development resource allowances?